From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Fri, 29 Dec 2023 13:05:45 +0100
Subject: [PATCH] Hide Javadoc lint warnings

Hides lint warnings printed when Javadocs are generated.
This is done because Paper code contains many instances that trigger these warnings, leading to them filling the terminal every time Javadocs are generated.

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Fiddle - https://fiddlemc.org

This patch is based on the following patch:
"Hide irrelevant compilation warnings"
By: Martijn Muijsers <martijnmuijsers@live.nl>
As part of: Gale (https://github.com/GaleMC/Gale)
Licensed under: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)

* Gale description *

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Gale - https://galemc.org

diff --git a/build.gradle.kts b/build.gradle.kts
index a7de1e92e49af2ac15f3e6c9bca9822a04116ca6..61fa6eb1f28c78a26ca3394f9e940ec5809f5f6c 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -155,6 +155,8 @@ tasks.withType<Javadoc> {
             into("build/docs/javadoc")
         }
     }
+
+    options.addStringOption("Xdoclint:none", "-quiet") // Fiddle - Gale - hide Javadoc lint warnings
 }
 
 tasks.test {
